### PR TITLE
Better rendering of generic attributes

### LINKED ISSUE #10: Display the attributes
Provide a function to display the simple attributes, can be inspired on json2html or json2table.

### PR DESCRIPTION
**Pull request by sbrunner**
Fix #10

### CODE REVIEW

File: pyramid_ogcapi/json2html.py [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+"""
+
+import html
+import json
+from typing import Optional, Union
+
+Json = Union[dict[str, "Json"], list["Json"], int, float, str, None, bool]
+
+
+def convert(json_input: Json, attributes_json_output: Optional[list[str]] = None, table_attributes=None):
```
Comment (vuilleumierc) [inline pyramid_ogcapi/json2html.py]: Missing return type?
Also about the parameter `attributes_json_output` -> isn't the output the HTML table (also in the doc string)?

File: pyramid_ogcapi/json2html.py [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+        Returns
+        -------
+        str
+            Table row of headers.
+        """
+        return "<tr><th>" + "</th><th>".join([html.escape(h) for h in headers]) + "</th></tr>"
+
+    @staticmethod
+    def _dict_to_html_attributes(d: Json) -> str:
+        r"""
```
Comment (vuilleumierc) [inline pyramid_ogcapi/json2html.py:158]:
```suggestion
        """
```

**Review by vuilleumierc — APPROVED** [review]
LGTM
Is there a reason not to import json2table directly?

File: pyramid_ogcapi/json2html.py (hunk 1 above)
Comment (sbrunner) [reply pyramid_ogcapi/json2html.py]: Thanks :-)

File: pyramid_ogcapi/json2html.py (hunk 2 above)
Comment (sbrunner) [reply pyramid_ogcapi/json2html.py:158]: Without that, I get the error:
[D301(pydocstyle), None] Use r""" if any backslashes in a docstring

**Comment by sbrunner** [comment]
Because:
- it's unmaintained
- it's relay simple
- I make some changes, like to render in JSON some part of the object.

**Pull request merged**